tests/test_address_cmd.py

```diff
@@ -0,0 +1,4 @@
+def test_address(cmd, button):
+    cmd.get_address(button=button, bip32_path="m/44'/280'/0'/0")
```

Comment:
We are not validating the return of the address. Is it possible to do?

Reply:
There is no return data and if the return code is not a `SW_OK` it should raise an exception failing the test.